### PR TITLE
Navbar tweak

### PR DESCRIPTION
**Pull request by codyloyd**

### CODE REVIEW

File: app/assets/stylesheets/components/logo.scss [hunk 1]
```diff
@@ -1,5 +1,6 @@
 .logo {
   a {
+    width: 100%;
```
Comment (arun1595) [inline app/assets/stylesheets/components/logo.scss]: Is this necessary?

Comment (codyloyd) [inline app/assets/stylesheets/components/logo.scss]: oh dang.. no.

I was trying something else that needed it and didn't get it deleted when I reverted. good catch

Comment (codyloyd) [inline app/assets/stylesheets/components/logo.scss]: common problem over here lol

fixed.

**Comment by KevinMulhern** [comment]
LGTM 👍

**Pull request merged**